internal/packager/init_test.go

```diff
@@ -44,6 +44,44 @@ services:
 	assert.Assert(t, fs.Equal(dir.Join(appName), manifest))
 }
 
+func TestInitFromComposeFileWithFlattenedParams(t *testing.T) {
+	composeData := `
+version: '3.0'
+services:
+  prometheus:
+    image: prom/prometheus:${versions.prometheus:-v2.7.0}
+    ports:
+      - 9090:9090
+
+  alertmanager:
+    image: prom/alertmanager:${versions.alertmanager:-v0.15.0}
+    ports:
+      - 9093:9093
+`
+	inputDir := fs.NewDir(t, "app_input_",
+		fs.WithFile(internal.ComposeFileName, composeData),
+	)
+	defer inputDir.Remove()
+
+	appName := "my.dockerapp"
+	dir := fs.NewDir(t, "app_",
+		fs.WithDir(appName),
+	)
+	defer dir.Remove()
+
+	err := initFromComposeFile(dir.Join(appName), inputDir.Join(internal.ComposeFileName))
+	assert.NilError(t, err)
+
+	manifest := fs.Expected(
+		t,
+		fs.WithMode(0755),
+		fs.WithFile(internal.ComposeFileName, composeData, fs.WithMode(0644)),
+		fs.WithFile(internal.ParametersFileName, "versions:\n  alertmanager: v0.15.0\n  prometheus: v2.7.0\n", fs.WithMode(0644)),
```

Comment:
nit: can you extract the expected value to a const string just above the `fs.Expected` call? And use the backquote notation?
```go
const expectedParameters = ` versions:
  alertmanager: v0.15.0
  prometheus: v2.7.0
`
manifest := fs.Expected( ...
```